.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,4 +1,33 @@
-
+<!--
+Thanks for sending a pull request!  Here are some tips for you:
+  1. If this is your first time, please read our [contributor guidelines](https://github.com/elyra-ai/community/blob/master/contributing.md)
+    1.1 Please follow the [7 rules for a great commit message](https://github.com/elyra-ai/community/blob/master/contributing.md#creating-a-pull-request)
+  2. If the PR is unfinished, leave it as 'DRAFT', add '[WIP]' in your PR title, e.g., '[WIP] Your PR title ...' or use the 'status:Work in Progress' label.
+  3. Be sure to keep the PR description updated to reflect all changes.
+  4. Please write your PR title to summarize what this PR proposes.
+  5. If this PR involves UI changes, please provide a screen capture (before/after) for a faster review.
+  6. Remember to add 'Fixes #ISSUE_NUMBER' to get the issue properly closed when the PR is merged
+  7. Set the proper milestone based on the target release for the issue
+-->
+
+### What changes were proposed in this pull request?
+<!--
+Please clarify what changes you are proposing. The purpose of this section is to outline the changes and how this PR fixes the issue.
+If possible, please consider writing useful notes for better and faster reviews in your PR. See the examples below.
```

Comment:
> See the examples below.

I don't see examples of useful notes below.  Should this clause get dropped?

Reply:
I believe this is the "main section" of the description describing your changes.

Reply:
So are there other files that will be included for this template?  The `ISSUE_TEMPLATE` is a directory, whereas `PULL_REQUEST_TEMPLATE` is a file - so I'm confused about what this is a "main section" for.  

Can you provide a screenshot since I suspect I'm not visualizing where "examples" are included when creating a PR.

Reply:
I have updated the description with up to date screenshot in both edit mode/preview mode.

Reply:
Thank you.  I guess what you mean by "See examples below" are the two points immediately following that statement:
```
1. If you refactor code that leads to class changes, showing the updated class hierarchy will help reviewers.
2. If there is design documentation, please add the link.
```
I was expecting to see example texts from PRs down below in the file - leading to my confusion that the "examples" were missing.
```suggestion
If possible, please consider writing useful notes for better and faster reviews in your PR.  For example...
```
and perhaps change from numeric bullets to dotted-bullets since order is not applicable?